File: generated_code_examples/java/classification/linear.java
public class Model {

    public static double[] score(double[] input) {
        return new double[] {subroutine0(input), subroutine1(input), subroutine2(input)};
    }
    public static double subroutine0(double[] input) {
        return ((((9.774126241420623) + ((input[0]) * (-0.41545978008486634))) + ((input[1]) * (0.9619661444337051))) + ((input[2]) * (-2.5028461157608604))) + ((input[3]) * (-1.0766107732916166));
    }
    public static double subroutine1(double[] input) {
        return ((((2.248771246116064) + ((input[0]) * (0.5239098915475155))) + ((input[1]) * (-0.3177027667958222))) + ((input[2]) * (-0.20333498652290763))) + ((input[3]) * (-0.9399605394445277));
    }
    public static double subroutine2(double[] input) {
        return ((((-12.022897487538232) + ((input[0]) * (-0.10845011146263966))) + ((input[1]) * (-0.6442633776378855))) + ((input[2]) * (2.7061811022837774))) + ((input[3]) * (2.01657131273614));
    }
}

File: generated_code_examples/fortran/classification/linear.f90
module Model
    implicit none
contains
    function score(input)
        implicit none
        double precision, dimension(:) :: input

        double precision, dimension(3) :: score
        score(:) = (/ 9.700311953536998d0 + input(1) * -0.4128360473754751d0 + input(2) * 0.9680426131053453d0 + input(3) * -2.498310603183548d0 + input(4) * -1.0723230787022542d0, 2.1575759475871163d0 + input(1) * 0.5400806228605453d0 + input(2) * -0.3245383349519669d0 + input(3) * -0.2034493200950831d0 + input(4) * -0.9338183426196143d0, -11.857887901124615d0 + input(1) * -0.12724457548509432d0 + input(2) * -0.6435042781533917d0 + input(3) * 2.7017599232786216d0 + input(4) * 2.006141421321863d0 /)
        return
    end function score
end module Model
